docs/core/preview3/tools/csproj.md

```diff
@@ -19,19 +19,80 @@ This document outlines the changes that were added to the csproj files as part o
 [MSBuild](https://github.com/Microsoft/MSBuild). For more information about general project file syntax and reference, 
 see [the MSBuild project file](https://docs.microsoft.com/visualstudio/msbuild/msbuild-project-file-schema-reference) documentation.  
 
+## Implicit package references
+The metapackage included is tied to the target framework. That means that for `netcoreapp1.0` a proper version of the `Microsoft.NETCore.App` metapackage is referenced. Same goes for `netcoreapp1.1` target framework as well as `netstandard1.x` target frameworks. 
+
+As far as the rest of the behavior is concerned, the tools will work as expected and most of the gestures will remain the same (for example, `dotnet restore`). 
+
+### Migrating a project
+If you have an existing reference to the metapackage in your project.json, the migration (both in `dotnet migrate` as well as Visual Studio 2017) will include that reference in your new csproj project. This will cause the tools to issue the following warning when you try to build your project:
+
+> A PackageReference for [metapackage ID] was included in your project. This package is implicitly referenced by the .NET SDK and you do not typically need to reference it from your project. For more information, see https://aka.ms/sdkimplicitrefs.
+
+This warning simply means that you probably want to remove that package reference from your project file. If you do keep it, the specified version of the metapackage will be used. 
+
+### Recommendations
+With this new feature, you might be wondering when you should specify a version of the metapackage in the project and when not. Here is the overall guidance:
```

Comment:
we need to start writing more concisely.  you might be wondering just adds words that don't add value. also, by reading your recommendations, there's not one single case where you should specify a version of the metapackage via the PackageReference property, right?

If the migration is not adding those anymore, I'd just provide the overall guidance as part of the parent section.